forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,404 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use chrono::{DateTime, Utc};
+use clap::Subcommand;
+use colored::*;
+use forest_blocks::tipset_json::TipsetJson;
+use forest_cli_shared::cli::CliOpts;
+use forest_rpc_client::{
+    chain_get_name, chain_head, node_ops::node_status, start_time, wallet_balance,
+    wallet_default_address,
+};
+use forest_shim::econ::TokenAmount;
+use forest_utils::misc::LoggingColor;
+
+use fvm_shared::clock::EPOCH_DURATION_SECONDS;
+use fvm_shared::{clock::ChainEpoch, BLOCKS_PER_EPOCH};
+use num::BigInt;
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+
+use super::Config;
+use crate::cli::handle_rpc_err;
+use crate::humantoken::TokenAmountPretty;
+
+#[derive(Debug, Subcommand)]
+pub enum InfoCommand {
+    Show,
+}
+
+#[derive(Debug)]
+pub struct NodeStatusInfo {
+    /// How far behind the node is with respect to syncing to head
+    pub behind: Duration,
+    /// Chain health is the percentage denoting how close we are to having
+    /// an average of 5 blocks per tipset in the last couple of
+    /// hours. The number of blocks per tipset is non-deterministic
+    /// but averaging at 5 is considered healthy.
+    pub health: f64,
+    /// epoch the node is currently at
+    pub epoch: ChainEpoch,
+    /// BaseFee is the set price per unit of gas (measured in attoFIL/gas unit) to be burned (sent to an unrecoverable address) for every message execution
+    pub base_fee: TokenAmount,
+    /// sync status information
+    pub sync_status: SyncStatus,
+    /// Start time of the node
+    pub start_time: DateTime<Utc>,
+    /// Current network the node is running on
+    pub network: String,
+    /// Default wallet address selected.
+    pub default_wallet_address: Option<String>,
+    /// Default wallet address balance
+    pub default_wallet_address_balance: Option<String>,
```

Comment:
You've added documentation for `base_fee`, but my initial comment was that a lot of the documentation is superfluous, highlighting this struct, and can probably be removed.

If your reader is expected to know what base fee is already, then you can leave this out.

Basically every comment here (aside from `base_fee` now, just restates the variable name underneath.
Check out this change - what information has been lost? I'd argue none - the struct is already called `NodeInfo`, so we know that the information relates to a node.

```suggestion
    pub epoch: ChainEpoch,
    pub base_fee: TokenAmount,
    pub sync_status: SyncStatus,
    pub start_time: DateTime<Utc>,
    pub network: String,
    pub default_wallet_address: Option<String>,
    pub default_wallet_address_balance: Option<String>,
```

Reply:
Removed superflous docs where applicable.